src/main/kotlin/blackjack/Blackjack.kt

```diff
@@ -25,9 +26,17 @@ class Blackjack {
 
     private fun processPlayerTurns(players: Players) {
         for (player in players.players) {
-            while (true) {
-                val answerInput = inputView.getPlayerRequestInput(player.name)
-                if (blackjackController.checkPlayerRequest(answerInput, player)) break
+            processSinglePlayerTurn(player)
+        }
+    }
+
+    private fun processSinglePlayerTurn(player: Player) {
+        var continueTurn = true
+        while (continueTurn) {
```

Comment:
flag가 생긴건 좋습니다 👍 
좀 더 심화해보면 내부에서 플래그 수정을 하기보단 바로 체크하는 방식도 별도의 지역변수도생성하지 않아도 되어괜찮을 것 같네요. 

```suggestion
        while (player.playable()) {
```

```suggestion
    while (blackjackController.playable(answerInput, player)) {
```